document_page/views/document_page.xml

```diff
@@ -95,12 +95,12 @@
         <field name="name">document.page.menu.form</field>
         <field name="model">document.page</field>
         <field name="arch" type="xml">
-            <form string="Document Page">
+            <form string="Document Page" create="0">
```

Comment:
Why create=0?

Reply:
This is the form view used from custom menuitems that you create through the wizard, IMO it doesn't make sense that you can create a new page from there, as it is supposed to be a wiki placed at convenience.

Reply:
OK, I don't have the context but I trust you

Reply:
See this:
![image](https://user-images.githubusercontent.com/23449160/53559791-405e0400-3b4b-11e9-829b-c4c551ebbf3e.png)

It does make sense to you a create button from a "static" wiki?